environment.mk

```diff
@@ -72,7 +72,8 @@ else
                          $(call get_python_package_version,xgboost) \
                          $(call get_python_package_version,lightgbm) \
                          $(call get_python_package_version,h5py) \
-                         $(call get_python_package_version,pyarrow)
+                         $(call get_python_package_version,pyarrow) \
+                         $(call get_python_package_version,grpcio)
```

Comment:
Is `gprcio==1.43` not working when installed with pip?

Reply:
Not for M1, Ray errors out importing it when installed via pip on M1 linking to https://docs.ray.io/en/master/ray-overview/installation.html#m1-mac-apple-silicon-support where it asks us to conda install it.